spec/features/user/edits/user_can_edit_name_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+describe "As a logged in user" do
```

Comment:
Beautiful story. I could read this to my non-existent grandchildren at bedtime it's so nicely written.

Reply:
hah! users love stories.